src/NzbDrone.Core/Organizer/FileNameBuilder.cs

```diff
@@ -518,12 +518,20 @@ private void AddMediaInfoTokens(Dictionary<string, Func<TokenMatch, string>> tok
                 mediaInfoSubtitleLanguages = string.Format("[{0}]", mediaInfoSubtitleLanguages);
             }
 
-            tokenHandlers["{MediaInfo Video}"] = m => mediaInfoVideo;
-            tokenHandlers["{MediaInfo Audio}"] = m => mediaInfoAudio;
+            var videoBitDepth = episodeFile.MediaInfo.VideoBitDepth.HasValue ? $"{episodeFile.MediaInfo.VideoBitDepth} Bit" : string.Empty;
+            var audioChannels = episodeFile.MediaInfo.AudioChannelPositions.IsNotNullOrWhiteSpace() ? episodeFile.MediaInfo.AudioChannelPositions.Split('/').Sum(s => decimal.Parse(s)).ToString() : string.Empty;
```

Comment:
Use CultureInvariant to handle the decimal point.
Also, you can add some fallback logic to convert AudioChannels (2,4,6,8 to 2.0, 4.0, 5.1, 7.1).
My suggestion would be to extract it to a separate method.